src/cmds/scripts/pbs_habitat.in

```diff
@@ -721,9 +722,14 @@ if [ -z "$PBS_HOME" ]; then
 	exit 1
 fi
 if [ ! -d "$PBS_HOME" ]; then
-	echo "*** $PBS_HOME directory does not exist"
-	echo "*** Run pbs_postinstall to create it"
-	exit 1
+	if [ -x "$PBS_EXEC/sbin/pbs_server" ]; then
+		component="server"
+	elif [ -x "$PBS_EXEC/sbin/pbs_mom" ]; then
+		component="execution"
+	else
+		echo "*** PBS will not create home in client installation"
+	fi
+	${PBS_EXEC}/libexec/pbs_postinstall $component $PBS_VERSION $PBS_EXEC $PBS_HOME "" "sameconf"
```

Comment:
This was the earlier implementation. After installation, if you want to create or modify PBS_HOME, we need to execute postinstall with the following inputs - 
pbs_postinstall ${component} ${version} ${PBS_EXEC} ${PBS_HOME}
It will not read these values from pbs.conf since postinstall assumes that it is the old file. 
This would not be required since the admin might have already given these inputs to the pbs.conf file. As you have mentioned it is better to raise a warning before performing this operation.